src/components/InputPassword/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './InputPassword.css';
+
+const InputPassword = props => {
+    return <input 
```

Comment:
from line 5 - 9 please implement 2 space indenting

Reply:
@Mubangizi this has been fixed.